src/transformers/models/yoso/modeling_yoso.py

```diff
@@ -0,0 +1,1690 @@
+# coding=utf-8
+# Copyright 2022 University of Wisconsin-Madison and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch YOSO model."""
+
+
+import math
+import os
+
+import torch
+import torch.utils.checkpoint
+from packaging import version
+from torch import nn
+from torch.nn import BCEWithLogitsLoss, CrossEntropyLoss, MSELoss
+
+from ...activations import ACT2FN
+from ...file_utils import (
+    add_code_sample_docstrings,
+    add_start_docstrings,
+    add_start_docstrings_to_model_forward,
+    replace_return_docstrings,
+)
+from ...modeling_outputs import (
+    BaseModelOutputWithPastAndCrossAttentions,
+    CausalLMOutputWithCrossAttentions,
+    MaskedLMOutput,
+    MultipleChoiceModelOutput,
+    QuestionAnsweringModelOutput,
+    SequenceClassifierOutput,
+    TokenClassifierOutput,
+)
+from ...modeling_utils import (
+    PreTrainedModel,
+    apply_chunking_to_forward,
+    find_pruneable_heads_and_indices,
+    prune_linear_layer,
+)
+from ...utils import logging
+from .configuration_yoso import YosoConfig
+
+
+logger = logging.get_logger(__name__)
+
+_CHECKPOINT_FOR_DOC = "uw-madison/yoso-4096"
+_CONFIG_FOR_DOC = "YosoConfig"
+_TOKENIZER_FOR_DOC = "NystromformerTokenizer"
+
+YOSO_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "uw-madison/yoso-4096",
+    # See all YOSO models at https://huggingface.co/models?filter=yoso
+]
+
+
+def load_cuda_kernels():
+    try:
+        from torch.utils.cpp_extension import load
+
+        def append_root(files):
+            src_folder = os.path.dirname(os.path.realpath(__file__))
+            return [os.path.join(src_folder, file) for file in files]
+
+        src_files = append_root(
+            ["fast_lsh_cumulation_torch.cpp", "fast_lsh_cumulation.cu", "fast_lsh_cumulation_cuda.cu"]
+        )
+
+        fast_lsh_cumulation = load("fast_lsh_cumulation", src_files, verbose=True)
```

Comment:
```suggestion
        load("fast_lsh_cumulation", src_files, verbose=True)
```
I think you shouldn't store the result since you are not using it.